test/tools/ossfuzz/abiV2FuzzerCommon.h

```diff
@@ -0,0 +1,88 @@
+#pragma once
+
+#include <libsolidity/interface/CompilerStack.h>
+#include <libyul/AssemblyStack.h>
+#include <liblangutil/Exceptions.h>
+#include <liblangutil/SourceReferenceFormatter.h>
+#include <libdevcore/Keccak256.h>
+
+namespace dev
+{
+namespace test
+{
+namespace abiv2fuzzer
+{
+class SolidityExecutionFramework
```

Comment:
I think this class is not really needed. All you need is an instance of `dev::solidity::CompilerStack`

Reply:
I would need a wrapper around `CompilerStack` to configure it, wouldn't I? Actually this code was borrowed from here, perhaps I should deduplicate/refactor.

https://github.com/ethereum/solidity/blob/develop/test/libsolidity/SolidityExecutionFramework.cpp

Reply:
I'll comment what can be removed here.

Reply:
Thank you!